src/tests/unit/tests/DetailsView/components/requirement-view-next-requirement-configuration.test.tsx

```diff
@@ -154,7 +154,7 @@ describe('RequirementViewNextRequirementConfiguration', () => {
         });
 
         it('returns Complete button when it is last requirement in mediumPassRequirementKeys', () => {
```

Comment:
nit: **mediumPass**RequirementKeys here (and above on line 144).

Reply:
oh I thought I fixed this in the merge! good catch! fixing now!